src/app.js

```diff
@@ -81,134 +33,59 @@ var run = function() {
 			}
 			window.localStorage[key] = content;
 		}
-		SOL_CACHE_FILE = fileKey(Object.keys(files)[0]);
+		editor.setCacheFile(Object.keys(files)[0]);
```

Comment:
Agreed, there's a ton more to clean up, rename, and clarify. For now, I tried to keep the names as similar as possible.